tests/integration_tests.rs

```diff
@@ -716,3 +716,13 @@ fn do_not_detect_different_syntax_for_stdin_and_files() {
         from_utf8(&cmd_for_stdin.get_output().stdout).expect("output is valid utf-8")
     );
 }
+
+#[test]
+fn show_all_mode() {
+    bat()
+        .arg("--show-all")
+        .arg("nonprintable.txt")
+        .assert()
+        .stdout("hello•world␊├──┤␍␀␇␈␛")
```

Comment:
I believe this actually demonstrates a bug in the implementation in this PR: there should be an *actual* line-feed character after `␊`, right?

Reply:
Hm, yes, indeed. I'll take a look tomorrow.

Reply:
OK, it might look like a hack, but I could hardcode inserting line-feed right after `␊` character in last position of line. What do you think?

Reply:
I mean something line that:

```rust
let line = replace_nonprintable(line_buffer, self.config.tab_width);
write!(handle, "{}", line)?;
if line.chars().last() == Some('␊') {
    writeln!(handle)?;
}
```

Reply:
Sounds good. I also don't see another option right now.